aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Host/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/aspnetcore:2.0.1
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2-stretch-slim
```

Comment:
can we use without `-stretch-slim`?

Reply:
Both should be ok, the Dockerfile above references the dotnet project template.